clickhouse/tests/test_unit.py

```diff
@@ -0,0 +1,68 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import mock
+import pytest
+from six import PY3
+
+from datadog_checks.clickhouse import ClickhouseCheck, queries
+
+from .utils import ensure_csv_safe, parse_described_metrics
+
+pytestmark = pytest.mark.unit
+
+
+def error(*args, **kwargs):
+    raise Exception('test')
+
+
+def test_error_query(instance):
+    check = ClickhouseCheck('clickhouse', {}, [instance])
+    check.log = mock.MagicMock()
+    del check.check_initializations[-2]
+
+    client = mock.MagicMock()
+    client.execute_iter = error
+    check._client = client
+
+    check.run()
+    check.log.error.assert_any_call('Error querying %s: %s', 'system.metrics', mock.ANY)
+
+
+@pytest.mark.parametrize(
+    'metrics, ignored_columns, metric_source_url',
+    [
+        (
+            queries.SystemMetrics.query_data['columns'][1]['items'],
+            {'Revision', 'VersionInteger'},
+            'https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/dbms/src/Common/CurrentMetrics.cpp',
+        ),
+        (
+            queries.SystemEvents.query_data['columns'][1]['items'],
+            set(),
+            'https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/dbms/src/Common/ProfileEvents.cpp',
+        ),
+    ],
+    ids=['SystemMetrics', 'SystemEvents'],
+)
+def test_current_support(metrics, ignored_columns, metric_source_url):
```

Comment:
```suggestion
def test_current_support(metrics, ignored_columns, metric_source_url):
```

`test_current_support` is not very obvious about what it does. Could you make this test name a bit more explicit (e.g. `test_latest_metrics_supported`) or add a docstring?